src/main/resources/com/microsoft/jenkins/containeragents/aci/AciContainerTemplate/help-privateIpAddress.html

```diff
@@ -0,0 +1,14 @@
+<div>
+    Select if the container instance should use a private ip address.
+    <p>
+    If it use a private ip address, then information about the vnet and the subnet are mandatory.
+    </p>
+    <p>
+    Important: Vnet and subnet has to be created before launching the Container agent and the subnet delegation must
```

Comment:
Should we add validation for this? and or fill from a dropdown / combo box?

Reply:
If it is okay for you, I would implement it later.